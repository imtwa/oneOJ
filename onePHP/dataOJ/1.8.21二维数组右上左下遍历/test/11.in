11 12
66 95 68 82 97 96 91 96 25 33 33 16
22 61 93 76 58 73 73 96 22 99 23 13
19 72 87 47 94 28 94 95 54 55 52 32
20 89 61 40 71 60 86 75 4 78 56 97
43 98 66 60 40 81 84 27 56 33 66 80
27 26 41 81 45 21 4 23 15 49 59 78
6 5 76 85 44 20 3 34 46 97 29 67
86 67 80 54 98 12 54 63 74 59 32 90
34 12 51 71 78 21 66 33 19 44 50 57
85 45 72 5 6 29 62 68 9 89 44 51
50 94 50 81 97 38 4 18 54 48 15 69
