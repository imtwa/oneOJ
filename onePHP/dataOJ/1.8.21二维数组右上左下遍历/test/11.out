66
95
22
68
61
19
82
93
72
20
97
76
87
89
43
96
58
47
61
98
27
91
73
94
40
66
26
6
96
73
28
71
60
41
5
86
25
96
94
60
40
81
76
67
34
33
22
95
86
81
45
85
80
12
85
33
99
54
75
84
21
44
54
51
45
50
16
23
55
4
27
4
20
98
71
72
94
13
52
78
56
23
3
12
78
5
50
32
56
33
15
34
54
21
6
81
97
66
49
46
63
66
29
97
80
59
97
74
33
62
38
78
29
59
19
68
4
67
32
44
9
18
90
50
89
54
57
44
48
51
15
69
